app/presenters/trainee_contact_details.rb

```diff
@@ -0,0 +1,40 @@
+class TraineeContactDetails
+  attr_reader :trainee
+
+  def initialize(trainee)
+    @trainee = trainee
+  end
+
+  def call
+    relevant_attributes.map { |k, v|
+      [I18n.t("presenters.TraineeContactDetails.attributes.#{k}"), v]
+    }.to_h
+  end
+
+private
+
+  def relevant_attributes
+    attrs = address_fields
+    attrs.merge(trainee.attributes.select { |k, _v| relevant_fields.include?(k) })
+  end
+
+  def address_fields
+    {
+      address: address_values,
+    }
+  end
+
+  def address_values
```

Comment:
Should this be in the model?

Reply:
As this is a presentational concern, i'm reluctant to add it to the model as it'll pave way for other presentational details to creep in (something we should avoid i think). This scenario requires the full address without the postcode, another might require the postcode to be included etc. I think for now it'd be best to let the consumer of the class define how they want to render/build up the data.